adapter/src/lib.rs

```diff
@@ -186,3 +186,35 @@ fn parse_tokens(params: &[(ParamType, &str)], lenient: bool) -> Result<Vec<Token
         .collect::<Result<_, _>>()
         .map_err(From::from)
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use byteorder::{BigEndian, ByteOrder};
+    use chrono::{TimeZone, Utc};
+    use primitives::merkle_tree::MerkleTree;
+    use std::convert::TryFrom;
+
+    #[test]
+    fn test_get_signable_state_root_hash_is_aligned_with_js_impl() {
```

Comment:
I've decided to test with the Heartbeat state root, that's also why I've included the byteorder ( I will move it to dev-dependencies)